File: app/src/main/java/com/ikem/nwodo/cryptonyte/Result.java
package com.ikem.nwodo.cryptonyte;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class Result {

    @SerializedName("data")
    @Expose
    private Data data;

    @SerializedName("status")
    @Expose
    private String status = null;

    public Data getData() {
        return data;
    }

    public void setData(Data data) {
        this.data = data;
    }

    public String getStatus() {
        return status;
    }

    public void setStatus(String status) {
        this.status = status;
    }
}

File: app/src/main/java/com/ikem/nwodo/cryptonyte/db/model/Result.kt
package com.ikem.nwodo.cryptonyte.db.model

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class Result {

    @SerializedName("data")
    @Expose
    lateinit var data: Data

    @SerializedName("status")
    @Expose
    var status: String? = null
}